### PR TITLE
[ALRT-1] : Added email as part of UserEntity

### PR DESCRIPTION
**Pull request by chandrashekhar-macrometa**
For saving events in DB we require useremail.
Currently UserEntity object does not have any attribute for email. This attribute is added as part of this change.

Also the current getUser API only uses demo tenant.

Hence a new API is created to take user and tenant as parameter

### CODE REVIEW

File: src/main/java/com/c8db/entity/UserEntity.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
     private Boolean active;
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/UserEntity.java:28]: @chandrashekhar-macrometa - You could also use `Lombok` for that class if you like.

**Review by StoyanTc — APPROVED** [review]
@chandrashekhar-macrometa - Please add some details to the PR's description.

File: src/main/java/com/c8db/entity/UserEntity.java (hunk 1 above)
Comment (chandrashekhar-macrometa) [reply src/main/java/com/c8db/entity/UserEntity.java:35]: Lombok is added to UserEntity

**Comment by grainier-macrometa** [comment]
@chandrashekhar-macrometa Use `[ALRT-1]` [ALRT-1] in PR title/description or in a commit description, so that it'll get automatically linked to the respective JIRA issue.

**Review by yehan-macrometa — APPROVED** [review]

**Review by grainier-macrometa — APPROVED** [review]

**Pull request merged**